playbooks/roles/kibana/templates/nginx.conf.j2

```diff
@@ -0,0 +1,47 @@
+upstream elasticsearch_server {
+   server 127.0.0.1:9200;
+}
+
+server {
+  # Kibana server, templated by ansible
+
+  {% if NGINX_ENABLE_SSL %}
+
+  listen {{ KIBANA_SERVER_NAME }}:{{ KIBANA_NGINX_PORT }};
+  listen {{ KIBANA_SERVER_NAME }}:{{ KIBANA_SSL_NGINX_PORT }};
+
+  ssl_certificate /etc/ssl/certs/{{ NGINX_SSL_CERTIFICATE|basename }};
+  ssl_certificate_key /etc/ssl/private/{{ NGINX_SSL_KEY|basename }};
+
+  {% else %}
+  listen {{ KIBANA_SERVER_NAME }}:{{ KIBANA_NGINX_PORT }};
+  {% endif %}
+
+  access_log {{ nginx_log_dir }}/kibana.access.log;
+  error_log {{ nginx_log_dir }}/kibana.error.log error;
+
+
+  # Set image format types to expire in a very long time
+  location ~* ^.+\.(jpg|jpeg|gif|png|ico)$ {
+      access_log off;
+      expires max;
+  }
+
+  # Elastic Search
+  location /e {
+      rewrite /e/(.*) /$1 break;
+      proxy_set_header X-Real-IP $remote_addr;
+      proxy_set_header X-Forwarded-For $http_x_forwarded_for;
+
+      proxy_redirect off;
+      proxy_pass http://elasticsearch_server;
+  }
+
+  # Kibana
+  location / {
+      root {{ kibana_app_dir }}/htdocs;
```

Comment:
It looks like indents switch between 2 and 4 spaces here.
Existing convention in this repo for these files is to use 2 space indents; can you ensure that is used consistently in these `.j2` files?